ghc-lib-gen/src/Main.hs

```diff
@@ -251,7 +457,85 @@ generateCabal = do
         ,"    main-is: Main.hs"
         ]
 
--- | Run Hadrian to build the things that the Cabal file needs
+-- | Produces a ghc-lib-parser Cabal file.
+generateGhcLibParserCabal :: IO ()
+generateGhcLibParserCabal = do
+    lib <- mapM readCabalFile cabalFileLibraries
+    bin <- (:[]) <$> readCabalFile cabalFileBinary
+    let askField from x = nubSort $ concatMap (`askCabalField` x) from
+    let askFiles from x = nubSort $ concatMap (`askCabalFiles` x) from
+
+    writeFile "ghc-lib-parser.cabal" $ unlines $ map trimEnd $
+        -- header
+        ["cabal-version: >=1.22"
+        ,"build-type: Simple"
+        ,"name: ghc-lib-parser"
+        ,"version: 0.1.0"
```

Comment:
I assume we then manually change the version?

Reply:
Yes.